<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="sw">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="le_nom">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_max">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nbr Max</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>401</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_sort">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_search">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QLineEdit" name="le_nom_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nbr Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_max_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>190</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nbr_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nbr</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_id">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>200</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_id_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>451</width>
        <height>301</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>220</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>140</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Activite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_cin">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_noom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom">
        <property name="geometry">
         <rect>
          <x>71</x>
          <y>137</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_activite">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_sexe">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>65</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Homme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femme</string>
         </property>
        </item>
       </widget>
       <widget class="QTextEdit" name="te_">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>90</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="de_">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>205</x>
          <y>179</y>
          <width>50</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Cellule</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_cell">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>176</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>401</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_sort_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>CIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prenom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Datee</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sexe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Activite</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_search_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>258</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="l_total">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>258</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>230</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_id_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom_2">
        <property name="geometry">
         <rect>
          <x>91</x>
          <y>147</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="de_1">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>190</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="te_1">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>100</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_sexe_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>230</y>
          <width>65</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Homme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femme</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="le_cin_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_activite_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>70</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>48</x>
          <y>150</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_noom_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Activite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>225</x>
          <y>183</y>
          <width>50</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Cellule</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_cell_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>180</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>200</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_id_d_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cellule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>detenu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
